sqlglot/dialects/vertica.py

```diff
@@ -0,0 +1,29 @@
+from sqlglot import exp, generator, tokens
+from sqlglot.dialects.dialect import Dialect
+from sqlglot.tokens import Tokenizer, TokenType
+
+
+class Vertica(Dialect):
+    class Tokenizer(tokens.Tokenizer):
+        QUOTES = ["'", '"']
+        IDENTIFIERS = ["`"]
+
+        KEYWORDS = {
+            **Tokenizer.KEYWORDS,
+            "INT64": TokenType.BIGINT,
+            "FLOAT64": TokenType.DOUBLE,
+            "VARCHAR": TokenType.STRING,
+        }
+
+    class Generator(generator.Generator):
+        TYPE_MAPPING = {
+            exp.DataType.Type.TINYINT: "INT64",
+            exp.DataType.Type.SMALLINT: "INT64",
+            exp.DataType.Type.INT: "INT64",
+            exp.DataType.Type.BIGINT: "INT64",
+            exp.DataType.Type.DECIMAL: "NUMERIC",
+            exp.DataType.Type.FLOAT: "FLOAT64",
+            exp.DataType.Type.DOUBLE: "FLOAT64",
```

Comment:
Can you link to any documentation showing that all these numeric types are INT64/FLOAT64? 

From what I [found here](https://www.vertica.com/docs/9.3.x/HTML/Content/Authoring/SQLReferenceManual/DataTypes/Numeric/INTEGER.htm), all these types seem supported